tutorials: make CORAL example consistent

wihobbs opened the pull request:
Problem: the CORAL interactive tutorial uses the username "elvis" in all places but one.

Solution: make the document consistent by removing the last reference to "hobbs17" and replacing with "elvis".

jameshcorbett reviewed (approved):
🥇

cmoussa1 reviewed (approved):
LGTM!

wihobbs commented:
After running `flux start make html` I get this diff:

```
The HTML pages are in _build/html.
hobbs@hobbs-mbp:~/flux-docs$ git status
On branch update-elvis
Your branch is up to date with 'origin/update-elvis'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   auto_examples/auto_examples_jupyter.zip
	modified:   auto_examples/auto_examples_python.zip
	modified:   auto_examples/index.rst

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	sg_execution_times.rst

no changes added to commit (use "git add" and/or "git commit -a")
```
```diff
hobbs@hobbs-mbp:~/flux-docs$ git diff auto_examples/index.rst
diff --git a/auto_examples/index.rst b/auto_examples/index.rst
index be13a05..24218aa 100644
--- a/auto_examples/index.rst
+++ b/auto_examples/index.rst
@@ -22,6 +22,7 @@ of using the Flux Python API.
 
     <div class="sphx-glr-thumbnails">
 
+.. thumbnail-parent-div-open
 
 .. raw:: html
 
@@ -40,6 +41,8 @@ of using the Flux Python API.
     </div>
 
 
+.. thumbnail-parent-div-close
+
 .. raw:: html
 
     </div>
```

I don't understand why elvis should be causing this, and I'm not comfortable adding compressed zipped folders to a PR until I understand what they're doing. The underlying files do not differ:

```
hobbs@hobbs-mbp:~/flux-docs$ diff auto_examples/example_job_submit_api.py ../example_job_submit_api.py 
hobbs@hobbs-mbp:~/flux-docs$ diff auto_examples/example_job_submit_api.ipynb ../example_job_submit_api.ipynb 
```

wihobbs commented:
you could say I don't "git" it

cmoussa1 commented:
I tried restarting the failed job to see if it would generate any kind of specific error:

```
Testing that you built the auto-examples, if this fails run flux start make html locally.
  😢️ Auto examples were not updated! Different size of index.rst:
  1497 vs. 1557
```

I wonder why `index.rst` is being affected by the change you made in `corals.rst`?

wihobbs commented:
I don't think these problems have anything to do with any changes:
```
hobbs@hobbs-mbp:~/flux-docs$ git status
**HEAD detached at upstream/master**
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   auto_examples/auto_examples_jupyter.zip
	modified:   auto_examples/auto_examples_python.zip
	modified:   auto_examples/index.rst

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	sg_execution_times.rst

no changes added to commit (use "git add" and/or "git commit -a")
```
That's off of `HEAD` at `upstream/master`.

Also worth noting that @jameshcorbett wasn't hitting this problem 2 days ago. Could the sphinx-gallery v0.17.1 release from 3 days ago have anything to do with this? When I looked more closely at the `flux start make html` commands, that's the only thing that occurred to me, although rolling back the version didn't solve the problem.

wihobbs commented:
Hmm. Interesting. My working theory is that some change was made to sphinx-gallery v0.17.1 that requires these new tags to make the gallery render.

jameshcorbett commented:
> Also worth noting that @jameshcorbett wasn't hitting this problem 2 days ago.

Not two days ago--I merged the PR two days ago but the action ran four months ago (it was an old PR I forgot to merge).

jameshcorbett reviewed (approved):
🥇

wihobbs commented:
Whoops. Ok, well, if we need to go back and fix we can. I don't think this really did anything drastic.
